docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss

```diff
@@ -1,3 +1,3 @@
-.hb-obj-page-width {
+.hb-page-width {
```

Comment:
there is a reference to this in `docroot/themes/humsci/humsci_basic/templates/block/block--local-tasks-block.html.twig` that needs to be changed